Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Ram/RamGraph.cs

```diff
@@ -82,6 +81,10 @@ void Update()
 
         public void UpdateParameters()
         {
+            if (m_shaderGraphAllocated == null) {
+                Init();
```

Comment:
I don't know if there's a better way than calling Init() here but it works for me.

Reply:
It's fine, I added the other ShaderGraphs as well, just in case :)